root/app/notification_base.py

```
@@ -67,18 +67,18 @@ def __init__(self, toml_path: str, sleep_time: int = 10) -> None:
                 "Notification configuration is missing in the TOML file."
             )
             return
-        apprise_config_path = notification_config.get("apprise_config_path")
-        if not apprise_config_path:
+        apprise_uri = notification_config.get("apprise_uri")
+        if not apprise_uri:
             ff_logging.log_failure(
-                "Apprise configuration path is missing in the TOML file."
+                "Apprise configuration URI is missing in the TOML file."
             )
             return
 
         self.apprise = apprise.Apprise()
-        added = self.apprise.add(apprise_config_path)
+        added = self.apprise.add(apprise_uri)
```

I think that right now, this is only going to work for an individual alert. Can you make this as a list instead so that people can add in whatever URLs they want?

---

let me test it real quick but I'm pretty sure multiple targets can be done as is using the apprise api server and an apprise api uri with tags.
eg. apprise://apprise.example.com/token?tags=email
that would send to all the notification services in apprise that have the tag email.

---

okay I tested 
apprise_uri = "apprise://localhost:8000/apprise?tags=discord"
and it successfully sends to all of my tagged notification urls in apprise.

I also tried a few methods of getting multiple uris in directly but for some reason the library does not like multiple urls being loaded in.
I've tried using the AppriseConfig function and I've tried adding them directly from a list like you asked but for some reason it just silently fails even with the debug argument set.